index.html

```diff
@@ -16,6 +16,7 @@
                     company: "Microsoft",
                     companyURL: "https://www.microsoft.com"}],
         github: "WICG/window-controls-overlay",
+        shortName:"window-controls-overlay",
```

Comment:
This should work too: 

```suggestion
        "github": "w3c/window-controls-overlay",
```